bundles/org.openhab.binding.automower/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -18,13 +18,11 @@
 					This application also needs to be connected to the
 					"Authentication API" and the "Automower Connect API"</description>
 			</parameter>
-			<parameter name="userName" type="text" required="true">
-				<label>User Name</label>
-				<description>The user name for which the application key has been issued</description>
-			</parameter>
-			<parameter name="password" type="text" required="true">
-				<context>password</context>
-				<description>The password for the given user</description>
+			<parameter name="appSecret" type="text" required="true">
+				<label>Application Secret</label>
+				<description>The Application Secret is required to communication with the Automower Connect Api at
```

Comment:
```suggestion
				<description>The Application Secret is required to communicate with the Automower Connect API at
```

Reply:
Thanks for checking, fixed [existing text](https://github.com/openhab/openhab-addons/pull/13263/files#diff-7c4fc533f8b23716c87311aae0f6095f1c88b7e526f10f11e4325a76abbc2217L14) also ...

Reply:
Thanks, I missed that. Now the I18N properties file must also be regenerated.

Reply:
Done.